gcomm/src/conf.cpp

```diff
@@ -125,19 +129,11 @@ gcomm::Conf::register_params(gu::Config& cnf)
 #define GCOMM_CONF_ADD(_x_) cnf.add(_x_, Flags::_x_);
 #define GCOMM_CONF_ADD_DEFAULT(_x_) cnf.add(_x_, Defaults::_x_, Flags::_x_);
 
-<<<<<<< HEAD
-    GCOMM_CONF_ADD (COMMON_BASE_HOST_KEY);
-    GCOMM_CONF_ADD (COMMON_BASE_PORT_KEY);
-#ifdef PXC
-    GCOMM_CONF_ADD (COMMON_BASE_DIR_KEY);
-#endif /* PXC */
-||||||| 63116854
-    GCOMM_CONF_ADD (COMMON_BASE_HOST_KEY);
-    GCOMM_CONF_ADD (COMMON_BASE_PORT_KEY);
-=======
     GCOMM_CONF_ADD (BaseHost);
     GCOMM_CONF_ADD (BasePort);
->>>>>>> codership/galera/4.x
+#ifdef PXC
+    GCOMM_CONF_ADD (CommonBaseDir);
```

Comment:
Cane we rename it to just `BaseDir`, just like `BaseHost` and `BasePort`?

Reply:
Done